docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -408,3 +402,260 @@ You see that the consumer component displays the stream of quote values on the w
++
+.Generating ConfigMap from service context
+[source,shell]
+----
+$ rhoas generate-config --type configmap --output-file ./rhoas-services.yaml
+----
++
+For a service context with a single Kafka instance, the contents of the `ConfigMap` file look like the following example:
++
+.Example ConfigMap file generated from service context
+[source,yaml,subs="+quotes"]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: _<service-context>_-configuration
+data:
+    ## Kafka Configuration
+    kafka_host: __<host>__:__<port>__
+----
++
+As indicated in the example, the name of the generated `ConfigMap` object has a format of `_<service-context>_-configuration`.
+
+. On the command line, create the `prices` topic required by the Quarkus application.
++
+[source,shell]
+----
+$ rhoas kafka topic create --name prices
+----
+
+. Create a service account for the Quarkus application to authenticate with your Kafka instance by performing the following actions.
+.. Create a new service account as an OpenShift secret and save this in a YAML file, as shown in the following example:
++
+[source,shell]
+----
+$ rhoas service-account create --file-format secret --output-file ./rhoas-secrets.yaml
+----
+.. Open the YAML file for the secret in your IDE.
++
+The contents of the file look like the following example:
++
+.Example secret file for service account credentials
+[source,yaml,subs="+quotes"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: service-account-credentials
+type: Opaque
+stringData:
+  RHOAS_SERVICE_ACCOUNT_CLIENT_ID: _<client-id>_
+  RHOAS_SERVICE_ACCOUNT_CLIENT_SECRET: _<client-secret>_
+  RHOAS_SERVICE_ACCOUNT_OAUTH_TOKEN_URL: _<oauth-token-endpoint-uri>_
+----
++
+As indicated in the example, the generated secret has a default name of `service-account-credentials`.
+
+.. Set Access Control List (ACL) permissions to enable the new service account to access resources in the Kafka instance.
++
+.Example command for granting access to Kafka instance
+[source,shell,subs="+quotes"]
+----
+$ rhoas kafka acl grant-access --producer --consumer --service-account _<client-id>_ --topic prices --group all
+----
++
+The command you entered allows applications to use the service account to produce and consume messages in the `prices` topic. Applications can use any consumer group and producer.
+
+.. To secure the sensitive credential information that is stored in the secret file, encode the client ID and client secret values from the file, as shown in the following commands:
++
+.Encoding credential information in Base64
+[source,shell,subs="+quotes"]
+----
+$ echo -n _<client-id>_ | base64
+$ echo -n _<client-secret>_ | base64
+----
++
+On the command line, you see Base64-encoded values of the client ID and client secret.
+
+.. In the secret file, overwrite the original client ID and client secret values with the corresponding Base64-encoded values shown on the command line. Save the secret file.
```

Comment:
Hey @jbyrne-redhat , the selected lines need to be omitted, on a final round of testing, encoding was found to be causing issues.

Cc @henryZrncik

Reply:
In given secret we can either use `stringData` and at most mention base64 encoding as beyond the scope,
or go with `Data` in which case inform user that not only he should encode with base64 2 of datas but it is necessary to encode all of them.

Reply:
Since CLI doesn't have support for `data`, I think it will be better if we don't mention it.

Reply:
Thanks, @rkpattnaik780 and @henryZrncik. I just did an end-to-end test of the example with those steps omitted, and it worked great :)